fancy-example/src/chat.rs

```diff
@@ -273,25 +279,25 @@ impl ChatExample {
                                 })
                                 .response;
 
-                            let points = if !is_message_from_myself {
-                                let top = response.rect.left_top() + Vec2::splat(margin);
+                            let points = if is_message_from_myself {
+                                let top = response.rect.right_top() + Vec2::new(-margin, margin);
                                 let arrow_rect =
-                                    Rect::from_two_pos(top, top + Vec2::new(-rounding, rounding));
+                                    Rect::from_two_pos(top, top + Vec2::new(rounding, rounding));
 
                                 vec![
                                     arrow_rect.left_top(),
                                     arrow_rect.right_top(),
-                                    arrow_rect.right_bottom(),
+                                    arrow_rect.left_bottom(),
                                 ]
                             } else {
-                                let top = response.rect.right_top() + Vec2::new(-margin, margin);
+                                let top = response.rect.left_top() + Vec2::splat(margin);
                                 let arrow_rect =
-                                    Rect::from_two_pos(top, top + Vec2::new(rounding, rounding));
+                                    Rect::from_two_pos(top, top + Vec2::new(-rounding, rounding));
 
                                 vec![
                                     arrow_rect.left_top(),
                                     arrow_rect.right_top(),
-                                    arrow_rect.left_bottom(),
+                                    arrow_rect.right_bottom(),
```

Comment:
What does this change?

Reply:
Just swapped the if else to make it easier to raead (clippy told me to!)

Reply:
Aaah I see, the way the diff is shown is a bit confusing